src/interdiff.c

```diff
@@ -58,6 +59,25 @@
 #define PATCH "patch"
 #endif
 
+/* ANSI color codes for diff output */
+#define COLOR_RESET     "\033[0m"
+#define COLOR_HEADER    "\033[1m"       /* Bold for headers */
+#define COLOR_FILE      "\033[1m"       /* Bold for filenames */
+#define COLOR_HUNK      "\033[36m"      /* Cyan for hunk headers */
+#define COLOR_ADDED     "\033[32m"      /* Green for added lines */
+#define COLOR_REMOVED   "\033[31m"      /* Red for removed lines */
+#define COLOR_CONTEXT   ""              /* No color for context lines */
+
+/* Line type for coloring */
+enum line_type {
+	LINE_CONTEXT,
```

Comment:
LINE_CONTEXT can be removed and instead of calling print_color() for LINE_CONTEXT, just fwrite() instead.